src/main/java/salesforce/ui/ToastMessage.java

```diff
@@ -0,0 +1,12 @@
+package salesforce.ui;
+
+/**
+ * Class for return a specific message text for Toast Message.
+ */
+public class ToastMessage {
+    private String quickTextSuccessfulCreated = "Quick Text \"%s\" was created.";
+
+    public String quickTextSuccessfulCreated(String quickTextName) {
```

Comment:
Im not sure.. what is the purpose of this class or method, should the text be extracted from the alert when its created?

Reply:
Yes, you extract the text of the alert, in this case from a created object, and it is the same in other features, for this in a different class
In the future it will be added for deletion or editing, which only changes the text of "Created, edit, deleted"